service-workers/service-worker/unregister-then-register.https.html

```diff
@@ -30,75 +30,24 @@
         });
   }, 'Unregister then register resolves to a new value');
 
-promise_test(function(t) {
-    var scope = 'resources/scope/re-register-while-old-registration-in-use';
-    var registration;
+promise_test(async function(t) {
+  const scope = 'resources/scope/re-register-while-old-registration-in-use';
+  const registration = await service_worker_unregister_and_register(t, worker_url, scope);
+  t.add_cleanup(() => service_worker_unregister(t, scope));
```

Comment:
It's nice to also remove the iframe in the cleanup before unregistering, as it helps unregister really unregister.